sycl/test-e2e/Basic/interop/make_kernel_subdevice_l0.cpp

```diff
@@ -0,0 +1,136 @@
+// REQUIRES: level_zero, opencl, level_zero_dev_kit
```

Comment:
Both

https://pipelines.actions.githubusercontent.com/serviceHosts/9326dcd2-c308-48b6-82dc-50179b80e5ba/_apis/pipelines/1/runs/97744/signedlogcontent/45?urlExpires=2023-09-18T20%3A47%3A42.9080356Z&urlSigningMethod=HMACV1&urlSignature=ez0F3KjhMUKjCUAguloOBF5pUMtjNTs

and

https://pipelines.actions.githubusercontent.com/serviceHosts/9326dcd2-c308-48b6-82dc-50179b80e5ba/_apis/pipelines/1/runs/97744/signedlogcontent/43?urlExpires=2023-09-18T20%3A48%3A09.1278284Z&urlSigningMethod=HMACV1&urlSignature=G3npqXGJEs11TuWX7h3OYEEGpubIA%2

have this test listed under UNSUPOORTED. I honestly can't imagine in what testing it's actually running. Can you please provide the link to where it passes?

Reply:
any follow-up on this?